phpunit.xml.dist

```diff
@@ -12,8 +12,7 @@
          failOnWarning="true"
          verbose="true">
 
-    <coverage cacheDirectory="build/.phpunit.cache/code-coverage"
```

Comment:
This file is the PHPUnit configuration we use, so it is better it has cache directory by default.